Makefile

```diff
@@ -55,7 +55,6 @@ generate: $(GO_FILES)
 	cp -r gentmp/github.com/bitnami-labs/sealed-secrets/pkg . && rm gentmp/ -rf
 
 manifests:
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0)
 	$(CONTROLLER_GEN) crd paths="./pkg/apis/..."  output:crd:artifacts:config=helm/sealed-secrets/crds/
```

Comment:
Why do we need to remove "go getting" the controller-gen binary?

Reply:
That is not defined and therefore doesn't work. Apart from that, we are not downloading any binaries on the Makefile.

Reply:
Fair enough